vars/pipelineSnapshotTesting.groovy

```diff
@@ -116,7 +116,10 @@ void call(Map config=[:]) {
                                 withCredentials([sshDevnetCredentials]) {
                                     sh label: "scp genesis.json from ${remoteServer}",
                                         script: """#!/bin/bash -e
-                                            scp -i \"\${PSSH_KEYFILE}\" \"\${PSSH_USER}\"@\"${remoteServer}\":/home/vega/.tendermint/config/genesis.json genesis.json
+                                            ssh -i \"\${PSSH_KEYFILE}\" -t \"\${PSSH_USER}\"@\"${remoteServer}\" 'sudo cp /home/vega/.tendermint/config/genesis.json /tmp/genesis.json && chmod a+rwx /tmp/genesis.json'
```

Comment:
let's make sure genesis.json does not exist before we try to copy it

Reply:
chmod needs to be called with `sudo` otherwise you will get the error:

```
chmod: changing permissions of '/tmp/genesis.json': Operation not permitted
```